src/lib/asn1/tests/TestASN1.cpp

```diff
@@ -158,7 +158,7 @@ static void TestASN1_Encode(nlTestSuite * inSuite, void * inContext)
     ASN1Writer writer;
     uint16_t encodedLen;
 
-    writer.Init(buf, sizeof(buf));
+    writer.Init(buf);
```

Comment:
Suggest adding a test for the following method in ASN1Reader/Writer:

```
void Init(const uint8_t (&data)[N])
    {
        Init(data, N);
    }
```

Should be trivial.

Reply:
Added new unit test that tests methods impacted by this PR.